app/destination/page.js

```diff
@@ -1,54 +1,85 @@
+    thumbnail: "/destination/image-europa.png",
+  },
+  {
+    id: 2,
+    name: "Moon",
+    description:
+      "Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers.",
+    thumbnail: "/destination/image-moon.png",
+  },
+  {
+    id: 3,
+    name: "Mars",
+    description:
+      "Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth.",
+    thumbnail: "/destination/image-mars.png",
+  },
+  {
+    id: 4,
+    name: "Titan",
+    description:
+      "Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.",
+    thumbnail: "/destination/image-titan.png",
+  },
+];
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
-
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+import { useState } from "react";
 
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import { PlanetWishlistItem } from "@/components/destination/PlanetWishlistItem";
 
 export const Destinations = () => {
-  const [selectedPlanets, onAddPlanet] = useState([]);
+  const [selectedPlanets, setSelectedPlanet] = useState([]);
+  // const [isPlanetSelected, setIsPlanetSelected] =
+  //   useState(false);
+  const [numberOfPlanets, setNumberOfPlanets] = useState(0);
 
-  let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  // let isPlanetSelected = false;
+  // let numberOfPlanets = 0;
 
   const onAddOrRemovePlanet = (name, index) => {
-    // TASK - React 1 week 2
-    // Implement this function
-    // If you press the "ADD PLANET" the selected planet should display "SELECTED"
-    // And the counter should update, how many planets are selected (numberOfPlanets)
-    console.log(`You seleceted the following planet: ${name}, with the index of ${index}`);
-  }
+    setSelectedPlanet((selectedPlanets) => {
```

Comment:
Nicely done!
If you have questions about this syntax just ask me on Sunday :)

Reply:
thank you for feed back